### ISSUE TITLE
Uncomment `export = wtf` ?

### ISSUE THREAD

**Issue by myrriad** [spencermountain/wtf_wikipedia#441]
I don't know if I should have opened this as an issue/bugfix/PR, but...

Is there a reason why `export = wtf` is commented out?

I was trying to figure out why the types weren't working in my environment (vscode, windows). I kept getting the error `Property 'fetch' does not exist on type 'typeof import("c:/[project-path]/node_modules/@types/wtf_wikipedia/index")` 

Trying to bugfix this was tough. Turns out, uncommenting a single line `// export = wtf` fixed all my problems. 

Take cytoscape's typings: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cytoscape/index.d.ts.
It starts out with a
`export = cytoscape;
export as namespace cytoscape;`

**Comment by spencermountain** [comment]
hey accyl, good question - I'm not sure.
gonna defer this one to @wvanderp  🤞?

**Comment by wvanderp** [comment]
Excellent timing. It is October again, so I'm working on opensource again. ( and also hunting for a t-shirt)

I see a few strange things here. You import the types from @types which is the DefinitelyTyped project which is deprecated. I think since we included the types in this repo.

also, I believe that that line is for creating a global var for importing a script into a browser which is a supported thing, but I need to read up on how this works

I will have a look at the types and the test for the types.

I hope that I have some time this weekend to look at it.

**Comment by spencermountain** [comment]
fixed in 9.0.2
thanks!

**Comment by rezonant** [comment]
I imagine it was commented out because this is not a valid export in ESM.

```
TS1203: Export assignment cannot be used when targeting ECMAScript modules. Consider using 'export default' or another module format instead.
    node_modules/wtf_wikipedia/types/index.d.ts:191:0:
      191 │ export = wtf
          ╵ ~~~~~~~~~~~~
```
